advanced-usage/index.md

```diff
@@ -266,15 +262,83 @@ excessive resources.
 {{ easy_image(src="system-info-kernel", width=600, class="pirate") }}
 {{ easy_image(src="system-info-about", width=600) }}
 
-### Network Test
-{{ service(service="Pardal", port=9120, link="/services/pardal", based=true) }}
+{% pirate() %}
+### Terminal
+{% end %}
+{{ service(service="ttyd" link="https://tsl0922.github.io/ttyd/", port=8088) }}
+{% pirate() %}
 
-The Network Test page measures real-time latency between BlueOS and the surface
-computer, and allows checking the upload and download speeds between them.
+The Terminal provides
+- A tmux session
+- Direct access into the core BlueOS docker container
+- Ready access to the tmux sessions of the core services (`CTRL+b s`)
+   - Useful for seeing logs as they update live
+   - Can kill services if necessary
+- Access to the underlying device via the `red-pill` utility
+   - Can return to the core container using the `exit` command, or pressing `CTRL+d`
+   - Can list available docker images (including extensions) with `docker image list`
+   - Can list active docker containers (including extensions) with `docker ps -a`
+{% end %}
```

Comment:
- You can also split horizontally `CTRL+b "`, vertically `CTRL+b %`, and use your cursor to resize the panels. For more advanced tips, checkout the [tmux cheat sheet website](https://tmuxcheatsheet.com/).